src/Microsoft.HttpRepl/Microsoft.HttpRepl.csproj

```diff
@@ -1,4 +1,4 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
I'm going to guess this is some kind of Encoding issue. I thought I had undone the "changes" VS noticed here but I guess they came back before I committed.